pages/settings/admin/devices.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <settings-page page-title="settings.devices.devices">
+  <settings-page
+    page-title="settings.devices.devices"
+    help-link="https://jellyfin.org/docs/general/server/devices.html"
```

Comment:
Would it be better to use an object in the `settings-page` component with every link, and just provide a key here? If for any reason, the links needed to be updated, it would be significantly easier to have them all in one place.

Reply:
Looks good to me. Which approach should we follow to check which page we're in?

Reply:
Up to you, we could even query the router instead. 

Something like below would be best for the settings-page
```
{
  'helpLink': 'https://...jellyfin docs...',
  'translationKey': 'settings.devices.devices'
}
```